rds_wav: add missign include for strcmp()

strcmp() is declared in string.h, so include it.

Signed-off-by: "Yann E. MORIN" <yann.morin.1998@free.fr>

diff -durN pifmrds-c67306ea9b8d827f45e0d90279d367e97119bcb1.orig/src/rds_wav.c pifmrds-c67306ea9b8d827f45e0d90279d367e97119bcb1/src/rds_wav.c
--- pifmrds-c67306ea9b8d827f45e0d90279d367e97119bcb1.orig/src/rds_wav.c	2014-05-04 18:21:40.000000000 +0200
+++ pifmrds-c67306ea9b8d827f45e0d90279d367e97119bcb1/src/rds_wav.c	2014-06-21 17:39:22.999128453 +0200
@@ -26,6 +26,7 @@
 #include <stdlib.h>
 #include <math.h>
 #include <sndfile.h>
+#include <string.h>
 
 #include "rds.h"
 #include "fm_mpx.h"
